release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -419,6 +419,26 @@ The Network Observability Operator releases updates independently from the {prod
 //[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
 //=== Cloud controller managers for additional cloud providers
 
+[discrete]
+[id="ocp-4-15-planned-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
+
+To prepare for this upcoming change, ensure that your workloads match the pod security admission profile that applies to them. Workloads that are not configured according to the enforced security standards defined globally or at the namespace level will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
```

Comment:
```suggestion
To prepare for this upcoming change, ensure that your workloads match the pod security admission profile that applies to them. If your workloads are not configured according to the enforced security standards defined globally or at the namespace level, they will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
```
Technically, your original sentence is correct, but I thought I'd offer a suggestion to make it a little more engaging by directly addressing the reader, as you did in the first sentence.